From af5dfe40aa17f8734c0fe0d585cff99e9b247f69 Mon Sep 17 00:00:00 2001
From: Richard Acayan <mailingradian@gmail.com>
Date: Tue, 6 Dec 2022 21:21:25 -0500
Subject: [PATCH] arm64: dts: qcom: sdm670-google-sargo: move modem firmware

Some of the firmware is loaded by the msm-firmware-loader and placed in
a different directory to avoid self-hosting it. Point to the modem
firmware loaded by the msm-firmware-loader.

Signed-off-by: Richard Acayan <mailingradian@gmail.com>
---
 arch/arm64/boot/dts/qcom/sdm670-google-sargo.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/qcom/sdm670-google-sargo.dts b/arch/arm64/boot/dts/qcom/sdm670-google-sargo.dts
index fedf693eafe9..714b350dd00d 100644
--- a/arch/arm64/boot/dts/qcom/sdm670-google-sargo.dts
+++ b/arch/arm64/boot/dts/qcom/sdm670-google-sargo.dts
@@ -606,7 +606,7 @@ &mdss_mdp {
 
 &mss_pil {
 	status = "okay";
-	firmware-name = "qcom/sdm670/sargo/mba.mbn", "qcom/sdm670/sargo/modem.mbn";
+	firmware-name = "msm-firmware-loader/target/mba.mbn", "msm-firmware-loader/target/modem.mdt";
 };
 
 &pm660l_gpios {
-- 
2.40.0

